src/OrchardCore.Modules/OrchardCore.Widgets/ResourceManifest.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.Extensions.Options;
+using OrchardCore.ResourceManagement;
+
+namespace OrchardCore.Widgets
```

Comment:
Please use file-scope namespaces. add `;` at the end of the line and i'll save indentations.